stories/bar/bar.stories.js

```diff
@@ -120,12 +152,12 @@ export const tabletAndMobile = () => `
 tabletAndMobile.parameters = {
     docs: {
         iframeHeight: 100,
-        storyDescription: 'For Tablet and Mobile (Cozy mode) apply the <code>fd-bar--cozy</code> class to the container element.'
+        storyDescription: 'The cozy bar contains a back button, page title, segmented button and product switch icon. It is responsive to both tablet and mobile screen sizes.<br>To apply cozy mode, add the <code>--cozy</code> modifier class to the container element.'
```

Comment:
is the icon on the far right a product switch icon? https://experience.sap.com/fiori-design-web/tool-header/#with-side-navigation-menu-icon-title-and-caret